modules/javafx.incubator.controls/src/main/java/com/sun/javafx/incubator/scene/control/MyImpl.java

```diff
@@ -0,0 +1,42 @@
+/*
```

Comment:
should the directory name be consistent with the rest of the modules?

that is,
'modules/incubator.controls' instead of
'modules/javafx.incubator.controls' ?

Reply:
The directory name is consistent with the rest of the modules.

Reply:
Using `modules/incubator.controls` would make it inconsistent.

Reply:
Ah, please disregard.
I was looking at it in eclipse - the project files are missing, so the name came out wrong.

Reply:
Yeah, I was going to mention that you need to add any IDE files you want. If you want to send / post the diffs for Eclipse, I'll update my branch.

Reply:
could you please cherry pick
097b373de2b3961bf9912b4b2d43b5672a288dc2

from
[ag.incubator.rich](https://github.com/andy-goryachev-oracle/jfx/tree/ag.incubator.rich) ?

Reply:
Done